### DISCUSSION TITLE
Select next button

### DISCUSSION THREAD

**Discussion by cricri-pingouin** [lwcorp/lwmenu#6] [Ideas]
In my somewhat similar but much lighter (much less customisable) implementation:
https://github.com/cricri-pingouin/SliRunDelete

I also have an option to automatically select the next button when clicking a button. Granted my app was designed for the actions to be executaed in order, whereas Lwmenu is much broader, so maybe an option to select any button? E.g. click any button and the close button becomes selected as default for next action.

**Comment by lwcorp** [comment]
>so maybe an option to select any button? E.g. click any button and the close button becomes selected as default for next action.

Yes, done and moved to issue #7 - could you comment there what do you think?

As for the rest, if you like to have the buttons run one after another, it's already supported [since v1.4.5](https://github.com/lwcorp/lwmenu/releases/tag/v1.4.5), if you arrange this manually via `buttonafter=X`: in button1 you can use `buttonafter=2`, in button2 you can use `buttonafter=3`, and so on.
If you like this to happen automatically with a global setting such as `buttonafter=*`, then feel free to submit it [in Issues](https://github.com/lwcorp/lwmenu/issues).
